source/develop/release-management/features/network/ipv6_subnets.html.md

```diff
@@ -164,9 +274,83 @@ provision the following networking API entities, represented in yaml:
       - subnet_id: "{{ ipv6_subnet.id }}"
 ~~~~~
 
+### Configuring stateless DHCPv6 on the subnet
+
+To configure stateless Dhcpv6 on the subnet, the user should provision
```

Comment:
I think if you could use a use-case diagram it would be more clear to the reader what's the flow.
Also please seperate setup with configuration, meaning it needs to be clear that I have to have a network ( I don't necessarily need to provision one), I have to have a router (?), and I need a logical port (?)

Reply:
I haven't added a diagram, but written step by step what needs to be done - and which possibilities are there to provision the networking API entities, and how to configure the guests.

Is it better now?

Reply:
Yes